src/nccl_ofi_rdma.c

```diff
@@ -4062,9 +4047,17 @@ static int listen(nccl_net_ofi_ep_t *base_ep,
 	l_comm->base.base.dev_id = dev_id;
 	l_comm->base.accept = accept;
 	l_comm->base.close = listen_close;
-	l_comm->tag = ep->tag;
 	l_comm->leader_local_ep = first_rail->ofi_ep;
 
+    /* Allocate listen communicator ID */
+	ret = nccl_ofi_idpool_allocate_id(ep->comm_idpool, &l_comm->comm_id);
+	if (OFI_UNLIKELY(ret != 0)) {
+		ret = ncclSystemError;
+		goto error;
+	}
+	id_allocated = true;
+	handle->tag = l_comm->comm_id;
```

Comment:
Can we rename `handle->tag` to `handle->comm_id`?

Reply:
agree